Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -21,6 +21,8 @@ namespace AzToolsFramework::ViewportUi::Internal
 {
     const static int HighlightBorderSize = 5;
     const static char* HighlightBorderColor = "#4A90E2";
+    const static int HighlightBorderBackButtonMargin = 5;
+    const static char* HighlightBorderBackButtonText = "X";
```

Comment:
suggestion: I think this ideally wants to be an icon, not text (we can reach out to Lee to get an icon). I think this could help with the styling to

Reply:
placeholder icon added 👍